lib/task_listener.js

```diff
@@ -181,6 +182,32 @@ export default class TaskListener extends EventEmitter {
               stack: e.stack
           });
         }
+        let stats = {
+          uptime: this.host.billingCycleUptime() || 0,
```

Comment:
If the only reason to do this is because the tests don't specify it, I would rather find a way to include in the tests rather than defaulting here.